Makefile

```diff
@@ -6,7 +6,9 @@ ifeq ($(STATIC),1)
     STATIC_LIB=true
 endif
 
-CFLAGS=-O3 -DNDEBUG
+CC ?= cc
+
+CFLAGS ?= -O3 -DNDEBUG
```

Comment:
In your Boron changes this is `CFLAGS += -O3 -DNDEBUG`.  Should we do that here as well?

Reply:
Probably yes. Will update PR.